ossm/merge_upstream.sh

```diff
@@ -67,8 +67,15 @@ main () {
     rm -rf bundle/**/*.yaml resources bundle.Dockerfile
     updateVersionsInOssmValuesYaml
     make gen
-    git add bundle resources chart bundle.Dockerfile "$HELM_VALUES_FILE"
+    git add bundle resources chart bundle.Dockerfile vendor "$HELM_VALUES_FILE"
```

Comment:
I'm not following this whole script... does it make sense to just have `git add .`?

Reply:
Not sure what was original idea. There should be no extra files so `git add .` should be safe. I will change it.

Reply:
It's fine, we can change to using `git add .` here but it's generally a bad practice IMO.